package.json

```diff
@@ -70,33 +77,27 @@
     "@now-ims/hapi-now-auth": "1.3.1",
     "blipp": "3.1.2",
     "bluebird": "3.5.3",
-    "@hapi/boom": "7.4.2",
+    "bulk-api": "file:src/bulkApi",
     "catbox": "10.0.6",
     "catbox-memory": "4.0.1",
     "commander": "2.19.0",
     "docdash": "1.0.3",
     "flat": "4.1.0",
     "glob": "7.1.3",
-    "@hapi/good": "8.2.0",
-    "@hapi/hapi": "18.3.1",
-    "@hapi/basic": "5.1.0",
     "hapi-auth-bearer-token": "6.1.1",
     "hapi-swagger": "9.4.2",
     "immutable": "3.8.2",
-    "@hapi/inert": "5.2.0",
-    "@hapi/joi": "15.0.3",
     "joi-currency-code": "2.0.2",
-    "@hapi/joi-date": "1.3.0",
     "jsdoc": "3.6.2",
     "knex": "0.16.3",
+    "leaked-handles": "^5.2.0",
```

Comment:
1. What are we using this for?
2. Remove carrot if we need it.

Reply:
It was used only for the unit tests (leaking mongoose handle) issue and it needs to be removed, or moved to devDependencies.